app/models/user.rb

```diff
@@ -94,6 +96,7 @@ class User < ActiveRecord::Base
 
   after_invitation_accepted :set_role_on_invitation_accept
   after_create :enable_notifications_for_agents
+  before_create :reject_invalid_characters_from_name
   acts_as_taggable_on :teams
```

Comment:
Could you change to before_save so people can't go in and add invalid characters later on?  I doubt they would, but right now they could